packages/template/vite/tmpl/vite.config.mjs

```diff
@@ -0,0 +1,4 @@
+import { defineConfig } from 'vite';
```

Comment:
question: what does this file do? I don't see it referenced in the `forge.config.js`

Reply:
**This is the biggest difference with Webpack templates!**
`vite.config.mjs` is the default configuration file for the Renderer process(Vite), which is identical to the behavior of Vite's project.
If we need to specify it explicitly, then it can be configured in `forge.config.js`.

```js
module.exports = {
  // ...
  plugins: [
    {
      name: '@electron-forge/plugin-vite',
      // ...
      config: {
        CLIOptions: {
          // vite.config.(js|mjs|cjs|ts|mts|cts)
          config: 'vite.config.mjs', // Default value
        },
      },
    },
  ],
};
```

Reply:
If we need stronger semantics, we can use `vite.renderer.config.mjs` and specify it in `forge.config.js`.
```js
CLIOptions: {
  config: 'vite.renderer.config.mjs',
}
```

Reply:
Makes sense, thank you!

Reply:
feat: semantic Renderer config 381cfba